vmdb/app/models/vm_redhat.rb

```diff
@@ -37,4 +37,12 @@ def self.calculate_power_state(raw_power_state)
     else             super
     end
   end
+
+  def supports_migrate?
+    false
+  end
+
+  def supports_reconfigure?
+    false
+  end
```

Comment:
@jrafanie i am not sure if template can be reconfigured or migrated. @gmcculloug can let me know.